static/app/components/modals/inviteMembersModal/index.spec.tsx

```diff
@@ -234,6 +235,66 @@ describe('InviteMembersModal', function () {
     expect(screen.getByRole('button', {name: 'Send invite'})).toBeDisabled();
   });
 
+  it('sends all successful invites with team default', async function () {
+    jest.mocked(useOrganization).mockReturnValue(org);
+    const createMemberMock = MockApiClient.addMockResponse({
+      url: `/organizations/${org.slug}/members/`,
+      method: 'POST',
+    });
+
+    render(<InviteMembersModal {...modalProps} />);
+
+    await waitFor(() => {
+      expect(screen.getByRole('button', {name: 'Add another'})).toBeInTheDocument();
+    });
+
+    // Setup two rows, one email each, the first with a admin role.
+    await userEvent.click(screen.getByRole('button', {name: 'Add another'}));
+
+    const emailInputs = screen.getAllByRole('textbox', {name: 'Email Addresses'});
+    const roleInputs = screen.getAllByRole('textbox', {name: 'Role'});
+
+    await userEvent.type(emailInputs[0], 'test1@test.com');
+    await userEvent.tab();
+    await selectEvent.select(roleInputs[0], 'Admin');
+
+    await userEvent.type(emailInputs[1], 'test2@test.com');
+    await userEvent.tab();
+
+    await userEvent.click(screen.getByRole('button', {name: 'Send invites (2)'}));
```

Comment:
Is there a simpler way to get into this state? i'm a little nervous that we have to chain a bunch of user events in this test, can make them flakey.

Reply:
I actually just copied this test from the one above haha!
I can try to see if there's a simpler way